src/server/routes/api/document/index.js

```diff
@@ -101,4 +120,11 @@ http.post('/query-reach', function( req, res ){
   .then( reachJson => res.json(reachJson) );
 });
 
+http.post('/convert-to-biopax', function( req, res ){
```

Comment:
Actually my initial idea was to avoid being have to fetch the document from database since we already have access the document from where this route would be called. However, it must be a trivial concern and changing it as a GET would be better as you mentioned. It will also give the flexibility of calling the route where document is not ready, if needed in the future.

Reply:
Do you prefer ``biopax`` or ``convert-to-biopax``? I think ``biopax`` would be too general if some other Biopax related api would be added in the future.

Reply:
I wouldn't be concerned about performance, as it's fast on the server side to query the DB.

Reply:
> Do you prefer biopax or convert-to-biopax? I think biopax would be too general if some other Biopax related api would be added in the future.

I think it's better to keep it simple and just use 'biopax' for now